scroll-animations-1/Overview.bs

```diff
@@ -542,25 +509,9 @@ as follows:
 1.  Otherwise, let <var>current scroll offset</var> be the current scroll offset of {{scrollSource}}
     in the direction specified by {{orientation}}.
 
-1.  If <var>current scroll offset</var> is less than {{startScrollOffset}}, return an unresolved
-    time value if {{fill}} is <code>none</code> or <code>forwards</code>, or 0 otherwise.
+1.  If <var>current scroll offset</var> is less than {{startScrollOffset}}, return 0.
```

Comment:
This is correct. However one thing to note is that in this situation the scroll timeline would be in the "InBefore" state which should cause the animation to be in the same state. So while timeline time is 0 but only animations with "fill:backwards" would be active.

At the moment we cannot specify this since `timeline.state` is not specified. But I think we should add a TODO here.

TODO(4750): Define what is the correct timeline state based on scroll offset.

Reply:
I summarized my understanding of the current plan here.

https://github.com/w3c/csswg-drafts/issues/4325#issuecomment-585295758